examples/rcbilling-demo/src/tests/main.test.ts

```diff
@@ -271,8 +270,50 @@ test.describe("Main", () => {
       await expect(page.getByText(title)).toBeVisible();
     });
   });
+
+  test.only("Tracks event SDK Initialized", async ({
```

Comment:
make sure to remove the `only` 

note: add a an eslint rule for this if there is none yet.

Reply:
Ah, just noticed there is one for `playwright.config.ts` but I guess that does not apply here.

Reply:
Actually the default of `vitest` is already to not allow this in CI.
<img width="458" alt="Screenshot 2025-01-27 at 10 26 59" src="https://github.com/user-attachments/assets/bcfa2efd-8b76-4bfc-8f1e-bf7430dacf7f" />

But this test is actually from the e2e suite and the config in `playwright.config.ts` takes care of it. Notice the build actually failed because of it.